.github/workflows/continuous-integration.yml

```diff
@@ -59,14 +59,20 @@ jobs:
           arguments: shellcheck
         if: runner.os == 'Linux'
       - name: Report jacoco coverage
+        id: jacoco_report
         uses: gradle/gradle-build-action@v2
         env:
           ORG_GRADLE_PROJECT_epApiVersion: ${{ matrix.epVersion }}
-          COVERALLS_REPO_TOKEN: ${{ secrets.COVERALLS_REPO_TOKEN }}
         with:
-          arguments: coveralls
+          arguments: :code-coverage-report:clean codeCoverageReport --no-build-cache
```

Comment:
Do we still need to prevent use of the build cache here?  Or was that just a transient error you were seeing?

Reply:
Just checked with the build cache and it seems to be working fine. Omitting accordingly.